config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -19,9 +19,6 @@ periodics:
         command:
         - "runner.sh"
         - "./scripts/ci-conformance.sh"
-        env:
-          - name: GINKGO_FOCUS
-            value: "Conformance Tests"
```

Comment:
not needed anymore https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1018/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R188